fieldtracing/fieldtracing.h

```diff
@@ -30,6 +30,9 @@
 #include "../fieldsolver/fs_common.h"
 #include "../fieldsolver/derivatives.hpp"
 #include "../sysboundary/ionosphere.h"
+#include "../logger.h"
+extern Logger logFile;
```

Comment:
This also needs logfile and didn't have it?

Reply:
was in the .cpp file, went to the .h file.